BuildLoader.py

```diff
@@ -394,6 +406,14 @@ def update_fit_table (self):
 		else :
 			addr = fit_address.value + (num_fit_entries + 1) * 16
 
+		# Add ACM3 with the reserved fit entry saved
+		if self._board.ACM3_SIZE > 0:
+			addr = self._board.ACM3_BASE
+			module_size = self._board.ACM3_SIZE
+			fit_entry = FitEntry.from_buffer(rom, fit_offset + (acm3_index+1)*16)
+			fit_entry.set_values(addr, module_size, 0x100, 0x3, 0)
```

Comment:
For spaceleft,
fit_entry.set_values(self._board.ACM3_BASE, module_size, 0x100, 0x3, 0)